wire-ios-request-strategy/Tests/Helpers/ActionHandlerTestBase.swift

```diff
@@ -69,9 +69,11 @@ class ActionHandlerTestBase<Action: EntityAction, Handler: ActionHandler<Action>
         for action: Action,
         expectedPath: String,
         expectedMethod: ZMTransportRequestMethod,
-        expectedData: Data,
-        expectedContentType: String,
-        apiVersion: APIVersion = .v1
+        expectedData: Data?,
+        expectedContentType: String?,
+        apiVersion: APIVersion = .v1,
+        file: StaticString = #filePath,
+        line: UInt = #line
```

Comment:
Make argument types optional, add `file` and `line` in order to show `XCTAssert`-warnings on the proper place.